### PR TITLE
Add commands.json files for redis modules

### PR DESCRIPTION
**Pull request by paoloredis**

### CODE REVIEW

File: data/commands_redisgraph.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
+{
```
Comment (dwdougherty) [inline data/commands_redisgraph.json:1]: RedisGraph is long dead, so this file isn't needed.

**Review by dwdougherty — APPROVED** [review]
Just remove commands_redisgraph.json; the module is long dead.

File: data/commands_redisgraph.json (hunk 1 above)
Comment (paoloredis) [reply data/commands_redisgraph.json:1]: Oh ok. So it's just the four modules?

Comment (dwdougherty) [reply data/commands_redisgraph.json]: Yes.

**Pull request merged**